src/pages/install/sign-in.tsx

```diff
@@ -27,7 +27,7 @@ export const InstalledSignIn: React.FC = () => {
       key="sign-in"
     >
       <Caption>Enter your 12 or 24 word Secret Key to continue.</Caption>
-      <Stack as="form" onSubmit={onSubmit} spacing="loose" mt="auto">
+      <Stack as="form" onSubmit={onSubmit} spacing="loose" mt={6}>
```

Comment:
we try to avoid using the number values for spacing, see a list of named literals here: https://github.com/blockstack/ui/blob/main/packages/ui-theme/src/literal-types.ts#L13 and values they map to here https://github.com/blockstack/ui/blob/main/packages/ui-theme/src/sizes.ts#L23

Reply:
Thanks for info, fixed.

Reply:
nice! this looks good, one other thing would be to ideally rebase your branch and squash the commits into 1 for a change like this.